packages/language-server/src/ui5-model.ts

```diff
@@ -11,33 +18,104 @@ import {
 } from "@ui5-language-assistant/semantic-model";
 import { Fetcher } from "../api";
 import { getLogger } from "./logger";
+import globby from "globby";
```

Comment:
put imports to 3rd party libs **before** relative / mono repo imports

Reply:
Done